docs/manual/templates.rst

```diff
@@ -319,3 +324,90 @@ Voorbeeld
          Neem alstublieft uw afspraakbevestiging mee.
 
 .. _Django defaultfilters reference: https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#built-in-filter-reference
+
+
+Registratie
+===========
+
+De registratie email is een optionele e-mail die wordt verzonden wanneer een formulier is geconfigureerd om de
+'registratie backend' te gebruiken. De registratie email heeft toegang tot alle
+gegevens uit het formulier en de waarden ingevuld door de gebruiker.
+
+**Speciale instructies**
+
+Dit zijn aanvullende variabelen en instructies die beschikbaar zijn voor het
+sjabloon. Als een variabele niet beschikbaar maar wel aanwezig is in het
+sjabloon, dan wordt deze niet getoond.
+
+==================================  ===========================================================================
+Variabele                           Beschrijving
+==================================  ===========================================================================
+``{{ form_name }}``                 De naam van het formulier.
+``{{ completed_on }}``              De tijdstip waarop het formulier werd ingezonden.
+``{{ public_reference }}``          De openbare referentie van de inzending.
+``{{ payment_received }}``          Indicatie of de gebruiker wel of niet heeft betaald.
+``{{ payment_order_id }}``          De referentie van de betaling.
+``{{ submission_language }}``       De taal van het formulier die werd ingezonden.
+``{{ co_signer }}``                 De voorletters en de achternaam van de persoon die het formulier heeft mede-ondertekend.
```

Comment:
Doesn't/should this now also include the identifying attribute (BSN) with Paul's latest improvements in #2716

Reply:
Yes!